### DISCUSSION TITLE
What do you think the goals of Instructor Training Checkout should be?

### DISCUSSION THREAD

**Discussion by klbarnes20** [carpentries/instructor-training#1433] [checkout-feedback]
### Background
When someone wants to be a certified Carpentries Instructor, they first take Instructor Training. After that, they must complete checkout to be certified.

**[Checkout](https://carpentries.github.io/instructor-training/checkout/) consists of three steps:**

- Attend a community discussion

- Make a contribution to a Carpentries lesson (including those in the incubator) or other resources

- Do a 5 minute teaching demo 

The Core Team is working on a project that is the first systematic review and opportunity to improve the checkout process since 2018. **We want to hear from you!**

**Comment by jevbelikov** [comment]
I think for the trainees, one of the goals is to try out some ways of participating in The Carpentries and another is to receive some initial training and experience in the Carpentries way of teaching and receiving/providing feedback. One can view certification as quality assurance, but a fairly lightweight one, rather an encouragement to try and to learn and improve.

**Comment by bencomp** [comment]
At a basic level, the Checkout is a signal to trainees that there is more to becoming an instructor than joining a two-day programme – even if the training already requires active participation. Completing the checkout steps should provide a feeling of accomplishment.

**Comment by bencomp** [comment]
In agreement with @jevbelikov, as a maintainer and instructor I find it beneficial that making a contribution to any lesson(s) demonstrates that the lesson materials can be improved by community members.

Sometimes, however, I feel such contributions should be more actionable.

Asking trainees to open an issue or even PR shortly after the training may help them look at the lesson with the pedagogical theory fresh in mind. There may be a few drawbacks to doing checkout shortly after the training:

- it is likely that the new instructor has no practical experience with the lesson, so reported issues are only based on "reading" the lesson
- it is sometimes hard to start a conversation with new instructors about their issue(s), if the submitter does not respond to comments
- some issues overlap with existing issues
- some issues include multiple unrelated things in the same issue

I do think we should still ask people to make a contribution, but perhaps the goal of making a contribution should (also) be to provide an actionable suggestion. Perhaps this could translate to:

- prefer suggestions for change over noting a problem
- prefer comments with suggestions on existing issues over new issues
- ask to react with an emoji to two issues, as well as making a comment/issue contribution

**Comment by bencomp** [comment]
Joining a community discussion helps to establish that the Instructor training is not just for the trainee, but that the trainee joins a community of fellow instructors, who have similar experiences and can help with questions about these experiences.

**Comment by sarasrking** [comment]
I agree that the checkout has a few things going on - it's a way for the community to see that a new instructor has some of the basics down for teaching (live coding, simple language, being inclusive, feedback) and also a way for the new instructor to get the keys to the community via GitHub and the other options for joining in and feeling invited to take on a role in the community. It can be overwhelming as a new starter though so any help we can offer could go a long way to helping people complete the steps.
